docs/bnb_greenfield/guide/getting-started/overview.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Overview
```

Comment:
Do not expect an `Overview`  in `getting-started`, let us just start.

Reply:
Done by removing the page /getting-started/overview.md